examples/forms.amp.html

```diff
@@ -72,7 +86,8 @@
 <form method="post"
       action="/form/html/post"
       action-xhr="/form/echo-json/post"
-      target="_blank">
+      target="_blank"
+      on="submit:lightbox1">
```

Comment:
browser not success. So this would happen regardless of the result of the submission but not when the submission doesn't actually happen (client-side validation errors would not trigger a submit event) .